### PR TITLE
feat: add gleam lsp

### PR DESCRIPTION
**Pull request by jayy-lmao**
For https://gleam.run.

Fairly new language, so may be too early to add. Just thought I may as well 🤷

### CODE REVIEW

**Comment by williambotman** [comment]
@mason-org/triage `notify(Reason::NewPullRequest)`

<sup>This comment ensures a notification is delivered to maintainers.</sup>

<sup>` 🤖 This is an automated comment. `  [` 📖 Source code `](https://github.com/williamboman/botman)</sup>

File: packages/gleam-ls/package.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    - target: darwin_arm64
+      file: gleam-v0.28.0-aarch64-apple-darwin.tar.gz
+      bin: gleam
+    - target: win_x64
+      file: gleam-v0.28.0-x86_64-pc-windows-msvc.zip
+      bin: gleam.exe
+    - target: win_arm64
+
+bin:
+  gleam lsp: "{{source.asset.bin}}"
```
Comment (jayy-lmao) [inline packages/gleam-ls/package.yaml:37]: I hope I did this part correctly. the `gleam lsp` command is what runs the lsp

**Comment by williamboman** [comment]
@williambotman /apply

```diff
diff --git a/packages/gleam-ls/package.yaml b/packages/gleam-ls/package.yaml
deleted file mode 100644
index 3dfc1ec..0000000
--- a/packages/gleam-ls/package.yaml
+++ /dev/null
@@ -1,37 +0,0 @@
----
-name: Gleam lsp
-description: |
-  A friendly language for building type-safe, scalable systems!
-homepage: https://gleam.run/
-licenses:
-  - Apache-2.0
-languages:
-  - Gleam
-categories:
-  - LSP
-
-source:
-  id: pkg:github.com/gleam-lang/gleam@v0.28.0
-  asset:
-    - target: linux_x64_gnu
-      file: gleam-v0.28.0-x86_64-unknown-linux-gnu.tar.gz
-      bin: gleam
-    - target: linux_arm64_musl
-      file: gleam-v0.28.0-aarch64-unknown-linux-musl.tar.gz
-      bin: gleam
-    - target: linux_x64_musl
-      file: gleam-v0.28.0-x86_64-unknown-linux-musl.tar.gz
-      bin: gleam
-    - target: darwin_x64
-      file: gleam-v0.28.0-x86_64-apple-darwin.tar.gz
-      bin: gleam
-    - target: darwin_arm64
-      file: gleam-v0.28.0-aarch64-apple-darwin.tar.gz
-      bin: gleam
-    - target: win_x64
-      file: gleam-v0.28.0-x86_64-pc-windows-msvc.zip
-      bin: gleam.exe
-    - target: win_arm64
-
-bin:
-  gleam lsp: "{{source.asset.bin}}"
diff --git a/packages/gleam/package.yaml b/packages/gleam/package.yaml
new file mode 100644
index 0000000..b03d5b2
--- /dev/null
+++ b/packages/gleam/package.yaml
@@ -0,0 +1,37 @@
+---
+name: gleam
+description: A friendly language for building type-safe, scalable systems!
+homepage: https://gleam.run/
+licenses:
+  - Apache-2.0
+languages:
+  - Gleam
+categories:
+  - Compiler
+  - Formatter
+  - LSP
+
+source:
+  id: pkg:github.com/gleam-lang/gleam@v0.28.0
+  asset:
+    - target: linux_x64_gnu
+      file: gleam-{{version}}-x86_64-unknown-linux-gnu.tar.gz
+      bin: gleam
+    - target: linux_x64
+      file: gleam-{{version}}-x86_64-unknown-linux-musl.tar.gz
+      bin: gleam
+    - target: linux_arm64
+      file: gleam-{{version}}-aarch64-unknown-linux-musl.tar.gz
+      bin: gleam
+    - target: darwin_x64
+      file: gleam-{{version}}-x86_64-apple-darwin.tar.gz
+      bin: gleam
+    - target: darwin_arm64
+      file: gleam-{{version}}-aarch64-apple-darwin.tar.gz
+      bin: gleam
+    - target: win_x64
+      file: gleam-{{version}}-x86_64-pc-windows-msvc.zip
+      bin: gleam.exe
+
+bin:
+  gleam: "{{source.asset.bin}}"
diff --git a/schemas/enums/language.json b/schemas/enums/language.json
index 9e8872a..15ec916 100644
--- a/schemas/enums/language.json
+++ b/schemas/enums/language.json
@@ -1,152 +1,152 @@
 {
-  "$schema": "https://json-schema.org/draft/2020-12/schema",
-  "$id": "https://github.com/mason-org/mason-registry/schemas/enums/language",
-  "type": "string",
-  "enum": [
-    ".NET",
-    "1С:Enterprise",
-    "AWK",
-    "Ada",
-    "Angular",
-    "Ansible",
-    "Antlers",
-    "Apex",
-    "Arduino",
-    "Assembly",
-    "Astro",
-    "Azure Pipelines",
-    "Bash",
-    "Bazel",
-    "Beancount",
-    "Bicep",
-    "Blade",
-    "BrighterScript",
-    "C",
-    "C#",
-    "C++",
-    "CMake",
-    "CQL",
-    "CSS",
-    "Clarity",
-    "Clojure",
-    "ClojureScript",
-    "CodeQL",
-    "Crystal",
-    "Csh",
-    "Cucumber",
-    "Cue",
-    "D",
-    "DOT",
-    "Dart",
-    "Dhall",
-    "Django",
-    "Docker",
-    "Drools",
-    "Elixir",
-    "Elm",
-    "Ember",
-    "Emmet",
-    "Erg",
-    "Erlang",
-    "F#",
-    "Fennel",
-    "Flow",
-    "Flux",
-    "Fortran",
-    "Gleam",
-    "Glimmer",
-    "Go",
-    "Gradle",
-    "GraphQL",
-    "Groovy",
-    "HAML",
-    "HTML",
-    "Handlebargs",
-    "Handlebars",
-    "Haskell",
-    "Haxe",
-    "Hoon",
-    "JSON",
-    "JSX",
-    "Java",
-    "JavaScript",
-    "Jinja",
-    "Jq",
-    "Jsonnet",
-    "Julia",
-    "Kotlin",
-    "Ksh",
-    "LESS",
-    "LaTeX",
-    "Lelwel",
-    "Liquid",
-    "Lua",
-    "Luau",
-    "Markdown",
-    "Metamath Zero",
-    "Mksh",
-    "Move",
-    "Mustache",
-    "Nginx",
-    "Nickel",
-    "Nim",
-    "Nix",
-    "Nunjucks",
-    "OCaml",
-    "OneScript",
-    "OpenAPI",
-    "OpenCL",
-    "OpenEdge",
-    "OpenFOAM",
-    "OpenSCAD",
-    "PHP",
-    "Perl",
-    "PowerShell",
-    "Prisma",
-    "Progress",
-    "Protobuf",
-    "Puppet",
-    "PureScript",
-    "Python",
-    "R",
-    "Raku",
-    "ReScript",
-    "Reason",
-    "Robot Framework",
-    "Ruby",
-    "Rust",
-    "SCSS",
-    "SQL",
-    "Salt",
-    "Scala",
-    "Sh",
-    "Shell",
-    "Slint",
-    "Smithy",
-    "Snakemake",
-    "Solidity",
-    "Sphinx",
-    "Starlark",
-    "Stylelint",
-    "Svelte",
-    "SystemVerilog",
-    "TOML",
-    "Teal",
-    "Terraform",
-    "Text",
-    "Twig",
-    "TypeScript",
-    "V",
-    "Vala",
-    "Veryl",
-    "VimScript",
-    "Visualforce",
-    "Vue",
-    "WGSL",
-    "XML",
-    "YAML",
-    "YARA",
-    "Zig",
-    "Zsh"
-  ]
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/mason-org/mason-registry/schemas/enums/language",
+    "type": "string",
+    "enum": [
+        ".NET",
+        "1С:Enterprise",
+        "AWK",
+        "Ada",
+        "Angular",
+        "Ansible",
+        "Antlers",
+        "Apex",
+        "Arduino",
+        "Assembly",
+        "Astro",
+        "Azure Pipelines",
+        "Bash",
+        "Bazel",
+        "Beancount",
+        "Bicep",
+        "Blade",
+        "BrighterScript",
+        "C",
+        "C#",
+        "C++",
+        "CMake",
+        "CQL",
+        "CSS",
+        "Clarity",
+        "Clojure",
+        "ClojureScript",
+        "CodeQL",
+        "Crystal",
+        "Csh",
+        "Cucumber",
+        "Cue",
+        "D",
+        "DOT",
+        "Dart",
+        "Dhall",
+        "Django",
+        "Docker",
+        "Drools",
+        "Elixir",
+        "Elm",
+        "Ember",
+        "Emmet",
+        "Erg",
+        "Erlang",
+        "F#",
+        "Fennel",
+        "Flow",
+        "Flux",
+        "Fortran",
+        "Gleam",
+        "Glimmer",
+        "Go",
+        "Gradle",
+        "GraphQL",
+        "Groovy",
+        "HAML",
+        "HTML",
+        "Handlebargs",
+        "Handlebars",
+        "Haskell",
+        "Haxe",
+        "Hoon",
+        "JSON",
+        "JSX",
+        "Java",
+        "JavaScript",
+        "Jinja",
+        "Jq",
+        "Jsonnet",
+        "Julia",
+        "Kotlin",
+        "Ksh",
+        "LESS",
+        "LaTeX",
+        "Lelwel",
+        "Liquid",
+        "Lua",
+        "Luau",
+        "Markdown",
+        "Metamath Zero",
+        "Mksh",
+        "Move",
+        "Mustache",
+        "Nginx",
+        "Nickel",
+        "Nim",
+        "Nix",
+        "Nunjucks",
+        "OCaml",
+        "OneScript",
+        "OpenAPI",
+        "OpenCL",
+        "OpenEdge",
+        "OpenFOAM",
+        "OpenSCAD",
+        "PHP",
+        "Perl",
+        "PowerShell",
+        "Prisma",
+        "Progress",
+        "Protobuf",
+        "Puppet",
+        "PureScript",
+        "Python",
+        "R",
+        "Raku",
+        "ReScript",
+        "Reason",
+        "Robot Framework",
+        "Ruby",
+        "Rust",
+        "SCSS",
+        "SQL",
+        "Salt",
+        "Scala",
+        "Sh",
+        "Shell",
+        "Slint",
+        "Smithy",
+        "Snakemake",
+        "Solidity",
+        "Sphinx",
+        "Starlark",
+        "Stylelint",
+        "Svelte",
+        "SystemVerilog",
+        "TOML",
+        "Teal",
+        "Terraform",
+        "Text",
+        "Twig",
+        "TypeScript",
+        "V",
+        "Vala",
+        "Veryl",
+        "VimScript",
+        "Visualforce",
+        "Vue",
+        "WGSL",
+        "XML",
+        "YAML",
+        "YARA",
+        "Zig",
+        "Zsh"
+    ]
 }
```

**Comment by williamboman** [comment]
@williambotman /apply

```diff
diff --git a/packages/gleam/package.yaml b/packages/gleam/package.yaml
index b03d5b2..ec9f17c 100644
--- a/packages/gleam/package.yaml
+++ b/packages/gleam/package.yaml
@@ -12,7 +12,7 @@ categories:
   - LSP
 
 source:
-  id: pkg:github.com/gleam-lang/gleam@v0.28.0
+  id: pkg:github/gleam-lang/gleam@v0.28.0
   asset:
     - target: linux_x64_gnu
       file: gleam-{{version}}-x86_64-unknown-linux-gnu.tar.gz
```

**Comment by williamboman** [comment]
@williambotman /apply

```diff
diff --git a/packages/gleam/package.yaml b/packages/gleam/package.yaml
index ec9f17c..9635172 100644
--- a/packages/gleam/package.yaml
+++ b/packages/gleam/package.yaml
@@ -14,9 +14,6 @@ categories:
 source:
   id: pkg:github/gleam-lang/gleam@v0.28.0
   asset:
-    - target: linux_x64_gnu
-      file: gleam-{{version}}-x86_64-unknown-linux-gnu.tar.gz
-      bin: gleam
     - target: linux_x64
       file: gleam-{{version}}-x86_64-unknown-linux-musl.tar.gz
       bin: gleam
```

**Pull request merged**